sql/telemetry/clients_aggregates_v1/query.sql

```diff
@@ -10,12 +34,23 @@ WITH scalar_aggregates AS (
       WHEN 'true' THEN sum(value)
     END AS agg_value
   FROM
-    clients_daily_scalar_aggregates_v1
+    clients_daily_scalar_aggregates_v1 AS scalar_aggs
   CROSS JOIN
     UNNEST(scalar_aggregates)
+  LEFT JOIN latest_versions
```

Comment:
Are `scalar_aggregates` being limited by version here? I see a join but no where clause.

Reply:
lol indeed. I missed this and just pushed it now. thanks!